include/gl_hsv_thresholder.h

```diff
@@ -60,7 +59,7 @@ class GlHsvThresholder {
     void returnBuffer(int fd);
     int testFrame(
         const std::array<GlHsvThresholder::DmaBufPlaneData, 3> &yuv_plane_data,
-        EGLint encoding, EGLint range, ProcessType type);
+        EGLint encoding, EGLint range, ProcessType type, bool useGrayScalePassThrough);
```

Comment:
we should pass this to the constructor instead, it's never gunna change during the lifetime of a gl hsv thresholder